apps/election-manager/src/lib/votecounting.ts

```diff
@@ -222,69 +222,80 @@ const buildVoteFromCvr = ({
 
 interface TallyParams {
   election: Election
-  precinctId?: string
-  scannerId?: string
   votes: VotesDict[]
+  filterContestsByParty?: string
 }
 
 export function tallyVotesByContest({
   election,
   votes,
+  filterContestsByParty,
```

Comment:
One thing worth considering: we may want to validate that this actually is a party id present in the election. We may also want to do this with precinct & ballot style.